tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -588,6 +598,10 @@ Status StorageManagerCanonical::array_metadata_consolidate(
         "Cannot consolidate array metadata; Array does not exist"));
   }
 
+  if (array_uri.is_tiledb()) {
```

Comment:
I agree, I'd like to remove it in a separate PR if that is okay?

Reply:
Sure, then in that separate PR please consider removing the leftover `Consolidator::check_array_uri` checks that I mentioned in that other comment since they are now redundant.